core/agents.py

```
@@ -77,3 +82,67 @@ def only_return_the_edited_content() -> str:
+    
+    Use the latest information available through web search to understand both products
+    and create an informed, relevant comparison angle.
+    """,
+    retries=2,
+    model_settings={"temperature": 0.8},
+)
+
+
+@competitor_vs_title_agent.system_prompt
+def add_competitor_vs_context(ctx):
+    from pydantic_ai import RunContext
```

Flake8 (7.3.0)

[error] 'pydantic_ai.RunContext' imported but unused

(F401)